### PR TITLE
Feature/images

### PR DESCRIPTION
**Pull request by accountas**
Server errorus tingejau atvaizuot listing creatione, tai gali silently failint teoriskai. For demo gal px

image
image

### CODE REVIEW

File: src/Components/CreateListing.js [hunk 1]
```diff
@@ -1,140 +1,130 @@
 import React, { useState } from 'react';
 import { useNavigate } from "react-router-dom";
-import DatePicker from 'react-datepicker'
```
Comment (ElektrosStulpas) [inline src/Components/CreateListing.js:3]: datepicker visiškai nedarom?

File: src/Components/CreateListing.js [hunk 2]
```diff
@@ -1,140 +1,130 @@
...
-                    startDate={startDate}
-                    endDate={endDate}
-                    selectsRange
-                    inline
-                />
-                <input type="button" value="Create the listing" onClick={handleSubmit} />
-            </form>
-            <button onClick={() => { navigate('/') }}>Cancel</button>
-            <PicUpload currentState={[tempState, setTempState]} handleChange={handleTempChange} />
+        //should probably move this form to separate component and reuse in edit listing
```
Comment (ElektrosStulpas) [inline src/Components/CreateListing.js:61]: yes, was thinking the same thing

File: src/Components/ListingCard.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react"
+import { Button, Card, Container, Row, Col, Form, Pagination } from 'react-bootstrap';
+
+export const ListingCard = (listing) => {
+    console.log(listing)
```
Comment (ElektrosStulpas) [inline src/Components/ListingCard.js:6]: prolly don't need this

**Review by ElektrosStulpas — APPROVED** [review]
LGTM

File: src/Components/CreateListing.js (hunk 1 above)
Comment (accountas) [reply src/Components/CreateListing.js:3]: Nelabai atm turim tam backendo + story

**Pull request merged**